### PR TITLE
Draft: Toast Notifications for Project Reunion

### PR DESCRIPTION
**Pull request by hulumane**
This spec covers Toasts for project reunion.

### CODE REVIEW

**Comment by andrewleader** [comment]
👍

File: specs/notifications-api/notifications-api.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+1. Declaration for xmlns:desktop
+1. In the IgnorableNamespaces attribute, com and desktop
+1. com:Extension for the COM activator using a new GUID of your choice. Be sure to include the Arguments="-ToastActivated" so that you know your launch was from a toast
+1. desktop:Extension for windows.toastNotificationActivation to declare your toast activator CLSID (the GUID from #4 above).
+
+Then, **in your app's startup code** (WinMain), subscribe to the **OnActivated** event.
+
+```csharp
+
+int main(int argc, char* argv[], char* envp[])
```
Comment (wjk) [inline specs/notifications-api/notifications-api.md:109]: Is this code C# or C++? That isn't what a main function in C# looks like, and WinMain is used in C++ apps only. If it's actually C#, it should be `Main(string[] args)`. Furthermore,  the `init_apartment()` call is C++-specific; to specify an STA apartment, put `[STAThread]` attribute on the Main method.

**Review by wjk — CHANGES_REQUESTED** [review]

File: specs/notifications-api/notifications-api.md (hunk 1 above)
Comment (hulumane) [reply specs/notifications-api/notifications-api.md:109]: Ah this is c++ sample code. Not c#. Will change to update that.

Comment (hulumane) [reply specs/notifications-api/notifications-api.md:109]: Good catch. I'll move this to c# syntax

**Pull request closed without merging**